pom.xml

```diff
@@ -43,10 +47,29 @@
         </license>
     </licenses>
 
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>http://www.eclipse.org/</url>
+    </organization>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/eclipse/microprofile-fault-tolerance/issues</url>
+    </issueManagement>
+
+    <developers>
+        <developer>
+            <name>Emily Jiang</name>
+            <url>https://github.com/Emily-Jiang</url>
+            <organization>IBM</organization>
+            <organizationUrl>https://www.ibm.com</organizationUrl>
+        </developer>
+    </developers>
+
     <scm>
-        <connection>scm:git:https://github.com/eclipse/microprofile-rest-client.git</connection>
-        <developerConnection>scm:git:https://github.com/eclipse/microprofile-rest-client.git</developerConnection>
-        <url>https://github.com/eclipse/microprofile-rest-client</url>
+        <connection>scm:git:https://github.com/eclipse/microprofile-fault-tolerance.git</connection>
+        <developerConnection>scm:git:https://github.com/eclipse/microprofile-fault-tolerance.git</developerConnection>
+        <url>https://github.com/eclipse/microprofile-fault-tolerance</url>
```

Comment:
same down here - should be `rest-client`, right?

Reply:
fixed